src/tests/base_kubeapi_test.py

```diff
@@ -165,3 +179,29 @@ def deploy_image_set(cls, cluster_name: str, api_client: ApiClient):
         image_set.create(openshift_release_image)
 
         return ClusterImageSetReference(image_set_name)
+
+    @classmethod
+    @JunitTestCase()
+    def apply_static_network_config(cls, kube_api_context: KubeAPIContext, nodes: Nodes, cluster_name: str):
+        if not global_variables.is_static_ip:
+            return None
+
```

Comment:
I would delete this (or delete the condition of the calling function), because changes in this condition should preferably happen on one place only

```suggestion
        if not global_variables.is_static_ip:
            return None

```